weave/compile_domain.py

```diff
@@ -37,12 +37,14 @@ def apply_domain_op_gql_translation(
 
     query_str_const_node = graph.ConstNode(types.String(), "")
     alias_list_const_node = graph.ConstNode(types.List(types.String()), [])
+    output_type_node = graph.ConstNode(types.TypeType(), types.Any())
```

Comment:
This is needed for mapping the GQL payload to weave (converting JSON blobs to strings, deserializing timestamps, so they dont have to be deserialized / mapped in gql_prop_op resolvers).